services/ndd_service.js

```diff
@@ -149,7 +149,7 @@ class NddService {
       nodePath += process.platform === 'win32' ? ';' : ':';
     nodePath += path.join(__dirname, '..', 'lib', 'preload');
     const env = {
-      NODE_OPTIONS: `--require ndb/preload.js`,
+      NODE_OPTIONS: `--require ${path.join(nodePath, 'ndb/preload.js')}`,
```

Comment:
I am not sure that this fix can resolve the problem. Do you have nice repro for this?
nodePath variable contains list of folders where node is looking for modules, this path plus ndb/preload.js most of the time would be something invalid.

Reply:
I am not sure if this fixes the other problem I run into with preload.js not found but if someone is already working on preload issues might be worth noting that I run into issues with preload.js not found on a new rails/webpacker setup... any basic out of the gate rails/webpacker install seems to exhibit the issue. Maybe this is the wrong place to mention it.